packages/dataviews/src/field-types/stories/index.story.tsx

```diff
@@ -766,3 +840,32 @@ export const NoType = ( {
 		<FieldTypeStory fields={ noTypeFields } type={ type } Edit={ Edit } />
 	);
 };
+
+export const TextWithPrefixSuffix = ( {
```

Comment:
Do we need this story? The new fields are going to be picked up by the text fields story. I'd prefer to keep the stories organized by field type.

Reply:
It was removed 👍